src/backend/src/services/change-requests.services.ts

```diff
@@ -429,6 +439,13 @@ export default class ChangeRequestsService {
       throwIfUncheckedDescriptionBullets(wbsElement.workPackage);
     }
 
+    const { changeRequests } = wbsElement;
```

Comment:
this code is copy and pasted please make it a util function and use it in both places